src/components/BorrowerProfile/LoanUse.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+	<span class="text-h2 text-gray-800">
```

Comment:
I'd recommend a <p> tag here as this is looks to be a.. paragraph, while <span> has no semantic meaning.

Minor, but .text-gray-800 is not needed in this case as that is the default color for all of our text.